js.go

```diff
@@ -1491,7 +1491,10 @@ func (js *js) subscribe(subj, queue string, cb MsgHandler, ch chan *Msg, isSync,
 		}
 
 		if js.opts.shouldTrace {
-			js.opts.trace(TraceSent, ccSubj, j, nil)
+			ctrace := js.opts.ctrace
+			if ctrace.RequestSent != nil {
+				ctrace.RequestSent(ccSubj, j, nil)
```

Comment:
The original approach had a single function (with operation type to distinguish), that's why we were passing nil as the last argument. Here, you have no reason to pass nil, just change the signature?

Reply:
Thanks just updated.